docs/feature-gates.md

```diff
@@ -310,3 +311,32 @@ Refer to this [document](service-loadbalancer.md) for more information.
 #### Requirements for this Feature
 
 This feature is currently only supported for Nodes running Linux.
+
+### ExternalNode
+
+The `ExternalNode` feature enables Antrea Agent runs on a Virtual Machine or a bare metal
+server which is not a Kubernetes Node.
+
+User creates ExternalNode object in the cluster to represent the Virtual Machine or the
```

Comment:
virtual machine

Reply:
I would not put many details in feature-gate.md. Maybe these two paragraphs should be moved to a separate doc, with a link in feature-gate.md.

We can simply say:

> The `ExternalNode` feature enables Antrea Agent runs on a virtual machine or a bare metal server which is not a Kubernetes Node, and enforces Antrea Network Policies for the VM/BM. For more information, please refer to...